README.md

```diff
@@ -19,9 +21,9 @@ Installation
 4. `node_modules/.bin/gulp`
 5. `bin/console assets:install --symlink`
 
-Running the site locally
+Docker Intallation - Running the site locally
 ------------------------
-The first time running the journal, you may need to build some dependant images first.
+<!-- The first time running the journal, you may need to build some dependant images first.
```

Comment:
These instructions are actually how to get it up and running with composer v2 when we had an issue with Dave's machine (before we knew that Nathan was using composer v1, and me and Dave v2). 

In otherwords - you could keep them, but label them as a workaround for composer v2.